build

```diff
@@ -1,54 +1,63 @@
 #!/bin/bash
-cd /tmp/rom # Depends on where source got synced
+cd /tmp/rom || exit                               # Depends on where source got synced
```

Comment:
What's the extra benefit if we use || exit here??

Reply:
sometime when codes dont get executed this will better exit the code its also from  shellcheck.net

Reply:
> sometime when codes dont get executed this will better exit the code its also from shellcheck.net

Mmm. Just checked shellcheck.net,
They are doing good work,, but a maintainer should know where rom is synced!